src/RFC-0121_ConsensusEncoding.md

```diff
@@ -225,33 +225,33 @@ pub struct OutputFeatures {
 }
 ```
 
-Output features consensus encoding is defined as follows (in order):
+[Output features](https://github.com/tari-project/tari/blob/development/base_layer/core/src/transactions/transaction_components/output_features.rs#L52) consensus encoding is defined as follows (in order):
 
 1. `version` - 1 unsigned byte. This should always be `0x00` but is reserved for future proofing.
-1. `maturity` - [varint]
-1. `flags` - 1 unsigned byte
-1. `metadata` - [dynamic vector]
-1. `unique_id` - [nullable] + [dynamic vector]
-1. `parent_public_key` - [nullable] + 32-byte compressed public key 
-1. `asset` - [nullable] + [AssetOutputFeatures](#assetoutputfeatures)
-1. `mint_non_fungible` - [nullable] + [MintNonFungibleFeatures](#mintnonfungiblefeatures)
-1. `sidechain_checkpoint` - [nullable] + [SideChainCheckpointFeatures](#sidechaincheckpointfeatures)
+2. `maturity` - [varint]
```

Comment:
The `1. 1. 1. ...` is markdown for a numbered list :)

```markdown
1. these 
1. are
1. all
1. 1s
```

Reply:
1. numbers
2. like
3. the
4. count
5. :)

Reply:
.. and now you have to manually update the numbers every time the list is edited..